packages/deployer/src/deployer.ts

```diff
@@ -58,10 +61,8 @@ export class Deployer {
      */
     public async deployAsync(contractName: string, args: any[] = []): Promise<Web3.ContractInstance> {
         const contractArtifactIfExists: ContractArtifact = this._loadContractArtifactIfExists(contractName);
-        const contractNetworkDataIfExists: ContractNetworkData = this._getContractNetworkDataFromArtifactIfExists(
-            contractArtifactIfExists,
-        );
-        const data = contractNetworkDataIfExists.bytecode;
+        const contractVersion: ContractVersion = this._getContractVersionFromArtifactIfExists(contractArtifactIfExists);
+        const data = contractVersion.compilerOutput.evm.bytecode.object;
```

Comment:
Again, we aren't guarenteed this will be available. Please check and handle the case where it isn't.